### PR TITLE
Move main package to root

### PR DESCRIPTION
**Pull request by avborup**
- Moves main.go to root
- Renames `internal` package to `custom`
- Updates Makefile

### CODE REVIEW

File: src/custom/logging.go [hunk 1]
```diff
@@ -64,7 +64,7 @@ func getLogLevel(level LogLevel) string {
 	}
 }
 
-// Used internally to print in the desired format
+// Used customly to print in the desired format
```
Comment (albe2669) [inline src/custom/logging.go:67]: Meaning kinda changes here

File: src/custom/error-handling.go [hunk 2]
```diff
@@ -35,7 +35,7 @@ func NewHiddenHttpErrorWithRelatedError(statusCode int, message string, err erro
 }
 
 func NewInternalServerError(err error) HttpError {
-	return NewHiddenHttpErrorWithRelatedError(500, "internal server error", err)
+	return NewHiddenHttpErrorWithRelatedError(500, "custom server error", err)
```
Comment (albe2669) [inline src/custom/error-handling.go:38]: Custom server errors?

**Review by albe2669 — CHANGES_REQUESTED** [review]

File: src/custom/logging.go (hunk 1 above)
Comment (avborup) [reply src/custom/logging.go:67]:
```suggestion
// Used internally to print in the desired format
```

File: src/custom/error-handling.go (hunk 2 above)
Comment (avborup) [reply src/custom/error-handling.go:38]:
```suggestion
	return NewHiddenHttpErrorWithRelatedError(500, "internal server error", err)
```

**Review by JoachimBorup — APPROVED** [review]

**Review by albe2669 — APPROVED** [review]

**Pull request merged**